nightfall-optimist/src/app.mjs

```diff
@@ -12,8 +19,18 @@ import {
   debug,
 } from './routes/index.mjs';
 
+const spec = swaggerJsDoc(options);
+const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
+logger.debug(`********* Optimist environment vars ${JSON.stringify(environment)}`); // TODO review logs
```

Comment:
We cant print this

Reply:
Have removed this [on the next PR](https://github.com/EYBlockchain/nightfall_3/pull/1225/files#diff-06bcbf60d087c9b88e8acb417f75e2b3c131d716431e5e36e4aa5dbfee1bdf61)

Reply:
This is a draft @druiz0992 we will clean it up once it's ready

Reply:
Just making sure we don’t forget